runtime/oti/j9nonbuilder.h

```diff
@@ -4219,6 +4222,7 @@ typedef struct J9CRIUCheckpointState {
 	UDATA libCRIUHandle;
 	struct J9VMInitArgs *restoreArgsList;
 	char *restoreArgsChars;
+	UDATA flags;
```

Comment:
`flags` fields shouldn't be UDATA sized as UDATA is 32 bits or 64 bits depending on the platform.  To not lose 32 bits worth of flags, be explicit on whether the field is a U32 or U64